### DISCUSSION TITLE
Maximizing Wi-Fi Throughput: Fine-Tuning Zephyr for Peak Performance with ESP32 SoCs in IoT Applications · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#227] [General]
# Maximizing Wi-Fi Throughput: Fine-Tuning Zephyr for Peak Performance with ESP32 SoCs in IoT Applications · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/zephyr-max-wifi-throughput/

**Comment by MateoSegura** [comment]
How would you get the WiFi stack to run on the "proc" CPU and my application on the "app" CPU?
